x-pack/plugins/integration_assistant/server/graphs/ecs/graph.ts

```diff
@@ -19,7 +16,7 @@ import { handleMissingKeys } from './missing';
 import { handleValidateMappings } from './validate';
 import { graphState } from './state';
 
-const handleCreateMappingChunks = async (state: EcsMappingState) => {
+const handleCreateMappingChunks = async ({ state }: EcsBaseNodeParams) => {
```

Comment:
Do we plan to extend the Params apart from state? Else we dan directly use `EcsMappingState` right?

Reply:
Do we not want to keep that consistent over all the nodes for now? It is in theory using EcsMapping state right now, just with the interface name?

Reply:
yeah , we do want to have it consistent. But was thinking did we miss anything to pass in other nodes?